torch/fx/passes/split_module.py

```diff
@@ -59,6 +59,7 @@ def split_module(
     keep_original_order: Optional[bool] = False,
     keep_original_node_name: Optional[bool] = False,
     keep_original_input_name: bool = True,
+    partition_affix: Optional[str] = None,
```

Comment:
Consider making this kwarg only

Reply:
Because the submod name in the end will be `submod_abc_i`, where `abc` is the added argument here. I don't want to give user an illusion that their new submod starts with `abc`.